cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +313,13 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@pytest.mark.automates(BZ(1528859))
```

Comment:
`BZ` is not needed there, and the example would be like `@pytest.mark.meta(automates=[1678130])`